src/domain/event/buttonPanel/ButtonPanel.tsx

```diff
@@ -0,0 +1,60 @@
+import { useField } from 'formik';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+
+import Button from '../../../common/components/button/Button';
+import Container from '../../app/layout/Container';
+import FormContainer from '../../app/layout/FormContainer';
+import { authenticatedSelector } from '../../auth/selectors';
+import { EVENT_FIELDS } from '../constants';
+import styles from './buttonPanel.module.scss';
+
+interface Props {
+  onSaveDraft: () => void;
+}
+
+const ButtonPanel: React.FC<Props> = ({ onSaveDraft }) => {
+  const { t } = useTranslation();
+  const [{ value: type }] = useField({ name: EVENT_FIELDS.TYPE });
+  const authenticated = useSelector(authenticatedSelector);
+
+  return (
+    <div className={styles.buttonPanel}>
+      <Container>
+        <FormContainer>
+          <div className={styles.buttonsRow}>
+            <div className={styles.draftButtonColumn}>
+              <Button
+                disabled={!authenticated}
+                fullWidth={true}
+                onClick={onSaveDraft}
+                title={
+                  !authenticated ? t('authentication.noRightsCreateEvent') : ''
```

Comment:
I would prefer to avoid negation for these but maybe it makes sense here?

Reply:
Changed to `authenticated ? '' : t('authentication.noRightsCreateEvent')`. I did similar change to publish button as well